README.md

```diff
@@ -676,6 +677,36 @@ $ where black
         - Working directory: `$ProjectFileDir$`
 	- Uncheck "Auto-save edited files to trigger the watcher"
 
+
+
+### Wing IDE
+
+1. Install `black`.
+
+```console
+$ pip install black
+```
+
+2. Make sure it runs from the command line, e.g.
+
+```console
+$ black --help
+```
+
+3. In Wing IDE, activate **OS Commands** panel  and define the command  **black** with e.g. the following values, to execute black on the currently selected file, using key binding **F1**:
+
+- right-click on a panel -> Insert Tool -> OS Commands
+- click on **+** in **OS Commands** -> New: Command line..
+  - Title: black
+  - Command Line: black %s
+  - I/O Encoding: Use Default 
+  - Key Binding: F1
+  - [x] Raise OS Commands when executed
+  - [x] Auto-save files before execution
+  - [x] Line mode
+
+4. Select a file in the editor and press **F1** , or whatever Key Binding you selected in step 3, to reformat the file.
```

Comment:
Should probably lowercase 'Key Binding', i.e. `whatever key binding you selected ...`.

Reply:
Also made this change.